aws/resource_aws_ses_domain_identity_policy.go

```diff
@@ -0,0 +1,140 @@
+package aws
+
+import (
+	"log"
+
+	"github.com/hashicorp/terraform/helper/schema"
+
+	"fmt"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/aws/aws-sdk-go/service/ses"
+	"github.com/hashicorp/terraform/helper/resource"
+)
+
+func resourceAwsSesDomainIdentityPolicy() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceAwsSesDomainIdentityPolicyCreate,
+		Read:   resourceAwsSesDomainIdentityPolicyRead,
+		Update: resourceAwsSesDomainIdentityPolicyUpdate,
+		Delete: resourceAwsSesDomainIdentityPolicyDelete,
+
+		Schema: map[string]*schema.Schema{
+			"arn": {
```

Comment:
This attribute should be changed to `identity` or `identity_arn` to match the expected value. If the API is always setting this to the ARN, we should likely name it `identity_arn` and validate it via:

```go
ValidateFunc: validateArn,
```